packages/minter-contracts/ligo/src/fixed_price_sale/fixed_price_sale_market_allowlisted.mligo

```diff
@@ -14,7 +14,6 @@ let check_allowed_param (param, allowlist : market_entry_points * allowlist) : u
   match param with
   | Sell p -> begin
       check_address_allowed (p.sale_token.fa2_address, allowlist, "SALE_ADDRESS_NOT_ALLOWED");
-      check_address_allowed (p.money_token.fa2_address, allowlist, "MONEY_ADDRESS_NOT_ALLOWED");
```

Comment:
If you are removing this, could you also update the `README` on this, and remove the `moneyAddressNotAllowed` constant from Haskell tests and the code that checks for this error?